### ISSUE TITLE
Update fail from 0.44.6 to 0.44.7

### ISSUE THREAD

**Issue by Peperehobbits01** [Voxelum/x-minecraft-launcher#661]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Launcher Version

0.44.7

### Operating System

Linux

### Architecture

x64

### Current Behavior

When trying to install whit the .deb package donwloaded from Github release, I get this error.
```
(Reading database ... 
(Reading database ... 5%%
(Reading database ... 10%%
(Reading database ... 15%%
(Reading database ... 20%%
(Reading database ... 25%%
(Reading database ... 30%%
(Reading database ... 35%%
(Reading database ... 40%%
(Reading database ... 45%%
(Reading database ... 50%%
(Reading database ... 55%%
(Reading database ... 60%%
(Reading database ... 65%%
(Reading database ... 70%%
(Reading database ... 75%%
(Reading database ... 80%%
(Reading database ... 85%%
(Reading database ... 90%%
(Reading database ... 95%%
(Reading database ... 100%%
(Reading database ... 378968 files and directories currently installed.)
Preparing to unpack .../xmcl-0.44.7-amd64.deb ...
Unpacking x-minecraft-launcher (0.44.7-832) ...
dpkg: error processing archive /home/alien-loic/Tlchargements/xmcl-0.44.7-amd64.deb (--install):
 trying to overwrite '/opt/X Minecraft Launcher/libEGL.so', which is also in package xmcl 0.44.6-826
dpkg-deb (subprocess): decompressing archive member: lzma write error: Broken pipe
dpkg-deb (subprocess): cannot copy archive member from '/home/alien-loic/Tlchargements/xmcl-0.44.7-amd64.deb' to decompressor pipe: failed to write (Broken pipe)
dpkg-deb: error: <decompress> subprocess returned error exit status 2
Errors were encountered while processing:
 /home/alien-loic/Tlchargements/xmcl-0.44.7-amd64.deb
```
And well it only hapened whit this files, because it dosen't hapened whit Discord. So yeah it's a bug but I don't no. How to fix it myself so here is the report xD.

### Expected Behavior

Instalation running nice and smooth, allowing me to use the latest version of the launcher. To have the best experience whit it as possible.

### Steps To Reproduce

1. Donwload the .deb files for amd64 in 64 bits
2. Double click on the .deb in your Nemo files manager of Linux Mint 20.3 Cinammon Edition.
3. Type your password as prompted
4. Then wait a few seconds and the install/update will fail. Getting you a beautiful error, the ones I have pute earlyer.

**Comment by ShadowAndNitrix** [comment]
Have You Tried Downloading The Launcher From The Website Instead Of Github?

**Comment by Peperehobbits01** [comment]
I have just tried but I get the same issues.

**Comment by SJK9906** [comment]
try to use appimage or use with arch on distrobox container (container on distrobox support GPU accelleration)

**Comment by SJK9906** [comment]
try to use appimage or use with arch/debian/fedora on distrobox container (container on distrobox support GPU accelleration)

**Comment by Love-in-idleness** [comment]
try it
https://mirrors.sdu.edu.cn/spark-store-repository/amd64-store/games/xmcl/xmcl_0.44.7-832_amd64.deb

**Comment by Peperehobbits01** [comment]
> try it https://mirrors.sdu.edu.cn/spark-store-repository/amd64-store/games/xmcl/xmcl_0.44.7-832_amd64.deb

Yeah sure, i'll give it a try.

**Comment by Peperehobbits01** [comment]
Well the problem is solved in the 0.44.8, so I will close this issues. Tho I do want to say when it's saying cleaned up startup code, god d'amm. It's just a small clean up xD
